docs/content/onedrive.md

```diff
@@ -402,13 +402,23 @@ It is a
 [known](https://github.com/OneDrive/onedrive-api-docs/issues/935#issuecomment-441741631)
 issue that Sharepoint (not OneDrive or OneDrive for Business) silently modifies
 uploaded files, mainly Office files (.docx, .xlsx, etc.), causing file size and
-hash checks to fail. To use rclone with such affected files on Sharepoint, you
+hash checks to fail. This will only happen for some files. To use rclone with such
+affected files on Sharepoint, you
```

Comment:
Because root cause is not well understood, the language on affected files is vague. This could be clarified with a link to the relevant OneDrive issue.